examples/gno.land/r/matijamarjanovic/config/config.gno

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"errors"
+	"std"
+)
+
+var (
+	main   std.Address // tony's main address
+	backup std.Address // backup address
```

Comment:
these two variable's names are seems ambiguous. Could you update their names? like `mainAddr`

Reply:
Hi, please see fix in this commit: 16529d3bbe583a13c406a3a2817ce12b37e8efd6